flocker/node/functional/test_docker.py

```diff
@@ -334,11 +366,13 @@ def test_pull_image_if_necessary(self):
         """
         The Docker image is pulled if it is unavailable locally.
         """
-        image = u"busybox:latest"
+        # Use an image that isn't likely to be in use by anything, since
+        # it's old, and isn't used by other tests:
+        image = u"busybox:ubuntu-12.04"
```

Comment:
:question: "isn't likely to be in use by anything" is a bit flakey; we could have a custom image that can be pulled and exists only for the specific purpose of being used by this test. That might be overkill though, up to you if you want to leave it as out-of-date busybox.